### ISSUE TITLE
Organizing our webpage

### ISSUE THREAD

**Issue by IlluminatiMan** [susw12/adscarrom#16]
We should start creating new files for the different pages and delete the tabs because we have to finish the beta version by *soon*. (I trust Aaron to do this because Sujay's doing the database/server and I'm coding the game.)

**Comment by susw12** [comment]
I agree with @IlluminatiMan on this one. Could @pie314159265 help us? @ALUKEM please finish this.

**Comment by susw12** [comment]
btw i got the server

**Comment by susw12** [comment]
I think 👍 ⚰️
